lib/main.js

```diff
@@ -18,43 +18,54 @@ function HTMLUglify(config) {
 
   var salt = config.salt || 'use the force harry';
```

Comment:
Well yeah, Math.random uses no system entropy at all. Why do we care about which is faster? This happens once...